src/ui/server/content-strings/pages/insurance/your-business/company/index.ts

```diff
@@ -1,10 +1,12 @@
 const COMPANY_DETAILS = {
   PAGE_TITLE: 'Your company details',
-  TRADING_NAME: 'Do you use a different trading name for this company?',
+  HAS_DIFFERENT_TRADING_NAME: 'Do you use a different trading name for this company?',
   TRADING_ADDRESS: 'Do you trade from a different address to your registered office address for this company?',
   WEBSITE: 'Enter your company website, if you have one (optional)',
   PHONE_NUMBER: 'Your UK telephone number (optional)',
   PHONE_NUMBER_HINT: 'We may need to contact you about your application',
+  DIFFERENT_TRADING_NAME: "What's your organisation's alternative trading name?",
+  ALTERNATIVE_TRADING_NAME_HINT: 'Your official trading name will still be on the policy.',
```

Comment:
Can we rename `ALTERNATIVE_TRADING_NAME_HINT` to `DIFFERENT_TRADING_NAME_HINT`?

Also at the top of the file hcnages, maybe `AlternativeTradingName`to `DifferentTradingName` to be consistent?